src/components/navItem.jsx

```diff
@@ -0,0 +1,11 @@
+// import React from 'react';
+
+// const NavItem = () => {
+//   const tabNames = ['Home', 'About', 'Connect', 'Donate'];
+//   let tabs = tabNames.map((name) => {
+//     return <a key={name} href={'/' + name }>{name}</a>
+//   });
+//   return tabs;
+// }
+
+// export default NavItem;
```

Comment:
Unused?

Reply:
Yep. I had the intent of eventually creating a nav with more to it than just My Cart, but I'll cross that bridge when I get there. Will delete for now for cleanliness' sake.